utils/consts.go

```diff
@@ -47,6 +47,7 @@ const (
 	jfrogWatchesEnv              = "JF_WATCHES"
 	jfrogProjectEnv              = "JF_PROJECT"
 	IncludeAllVulnerabilitiesEnv = "JF_INCLUDE_ALL_VULNERABILITIES"
+	KeepPreviousCommentsEnv      = "JF_AVOID_OLD_PR_COMMENTS_DELETION"
```

Comment:
Again, consustency is important - 

KeepPreviousCommentsEnv --> AvoidPreviousPrCommentsDeletionEnv

The same guideline goes for the code in `params.go`.